gradlew

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
+#!/bin/sh
```

Comment:
Not sure we should update the scripts here as this is a security risk and rarely necessary. We do have a github action to verify the files, so maybe it is fine, but then we should use the latest version: https://github.com/onionshare/onionshare-android/blob/main/.github/workflows/build.yml#L13